packages/linux-firmware/linux-firmware.spec

```
@@ -0,0 +1,89 @@
+%global debug_package %{nil}
+
+%global _firmwarepath   /usr/lib/firmware
```

This should be under the sysroot path, otherwise it will conflict with our `/usr` symlink at build time.

```suggestion
%global fwdir %{_cross_libdir}/firmware
```